### PR TITLE
fixed wrong midi export

### PR DESCRIPTION
**Pull request by handrok**
[midi_export_bug.gp.zip](https://github.com/user-attachments/files/16747703/midi_export_bug.gp.zip)

### CODE REVIEW

**Comment by Jojo-Schmitz** [comment]
Probably needed for 4.4.0 too

File: src/engraving/compat/midi/compatmidirender.cpp [hunk 1]
```diff
@@ -80,7 +80,7 @@ void CompatMidiRender::createPlayEvents(const Score* score, Measure const* start
                 }
 
                 Chord* chord = toChord(item);
-                Chord* nextChord = CompatMidiRender::getChordFromSegment(seg->next(st), track);
```
[the remark this answers is not in the document]
Comment (handrok) [reply src/engraving/compat/midi/compatmidirender.cpp:83]: It is two different method with different behaviour. `getChordFromSegment` returns chord from segment. But `chord->next();` returns next chord.

**Review by alexpavlov96 — APPROVED** [review]

**Comment by handrok** [comment]
> Probably needed for 4.4.0 too

It is old bug and it is not blocker for 4.4.0 release.

**Comment by Jojo-Schmitz** [comment]
But harmless enough?

**Review by alexpavlov96 — APPROVED** [review]

**Pull request merged**